scripts/pnodespy.py

```diff
@@ -37,40 +40,33 @@ def do_class_kinds():
 
 def do_iirs_subprg():
     classname = "vhdl__nodes"
-    print()
-    print("Get_Kind = {0}.{1}__get_kind".format(libname, classname))
-    print("Get_Location = {0}.{1}__get_location".format(libname, classname))
+    print(dedent("""
+        Get_Kind = {libname}.{classname}__get_kind
+        Get_Location = {libname}.{classname}__get_location
+        """).format(libname=libname, classname=classname)
+    )
     for k in pnodes.funcs:
-        print()
-        print(
-            "Get_{0} = {1}.{2}__get_{3}".format(
-                k.name, libname, classname, k.name.lower()
-            )
-        )
-        print()
-        print(
-            "Set_{0} = {1}.{2}__set_{3}".format(
-                k.name, libname, classname, k.name.lower(), k.pname, k.rname
-            )
+        print(dedent("""
+            Get_{kname} = {libname}.{classname}__get_{kname_lower}
+            Set_{kname} = {libname}.{classname}__set_{kname_lower}
+            """).format(kname=k.name, kname_lower=k.name.lower(), libname=libname, classname=classname)
         )
 
 
 def do_libghdl_elocations():
     classname = "vhdl__elocations"
-    print("from pyGHDL.libghdl import libghdl")
-    print()
+    print(dedent("""\
+        # Auto generated Python source file from Ada sources
+        # Call 'make' in 'src/vhdl' to regenerate:
```

Comment:
`xtools` does not exist now. Scripts are `scripts/pnodes.py` and `scripts/pnodespy.py`. I suggest rewording to: `Use scripts/pnodes*.py via make to regenerate:`